### PR TITLE
[enhancement] update argo workflows based on blog comments

### PR DESCRIPTION
**Pull request by season1946**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

### More

- [x] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/awslabs/data-on-eks/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR to add a new Add-on for [Terraform EKS Blueprints](https://github.com/aws-ia/terraform-aws-eks-blueprints) repo (if applicable)
- [ ] Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [x] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: schedulers/terraform/argo-workflow/helm-values/aws-for-fluentbit-values.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  enabled: true
+  match: "*"
+  region: ${region}
+  logGroupName: ${aws_for_fluent_bit_cw_log}
+  logStreamName:
+  logStreamPrefix: "fluentbit-"
+  logKey:
+  logFormat:
+  roleArn:
+  autoCreateGroup: true
```
Comment (vara-bonthu) [inline schedulers/terraform/argo-workflow/helm-values/aws-for-fluentbit-values.yaml:38]: change this to `autoCreateGroup: false`

File: schedulers/terraform/argo-workflow/argo-events/sensor-sqs-sparkjobs.yaml [hunk 2]
```diff
@@ -24,8 +24,20 @@ spec:
                 namespace: argo-workflows
               spec:
                 arguments: {}
-                entrypoint: sparkapp-operator
+                entrypoint: parallel-jobs
                 templates:
+                - name: parallel-jobs
+                  steps:
+                  - - name: hello1
```
Comment (vara-bonthu) [inline schedulers/terraform/argo-workflow/argo-events/sensor-sqs-sparkjobs.yaml:31]: Is this ok with two hyphens?

File: schedulers/terraform/argo-workflow/argo-events/sensor-sqs-sparkjobs.yaml [hunk 3]
```diff
@@ -24,8 +24,20 @@ spec:
...
+                      template: whalesay
+                      arguments:
+                        parameters: [{name: message, value: "spark-start!"}]
+                  - - name: spark-work
```
Comment (vara-bonthu) [inline schedulers/terraform/argo-workflow/argo-events/sensor-sqs-sparkjobs.yaml:35]: same here with two hyphens?

**Review by vara-bonthu — CHANGES_REQUESTED** [review]
Few comments to address otherwise it's good 👍🏼

File: schedulers/terraform/argo-workflow/helm-values/aws-for-fluentbit-values.yaml (hunk 1 above)
Comment (season1946) [reply schedulers/terraform/argo-workflow/helm-values/aws-for-fluentbit-values.yaml:38]: changed to false

**Review by vara-bonthu — COMMENTED** [review]
few more changes to be udpated

**Review by vara-bonthu — APPROVED** [review]
Thanks! LGTM 👍🏼

**Pull request merged**